src/hotspot/share/classfile/modules.cpp

```diff
@@ -695,11 +695,12 @@ const char* Modules::get_addmods_names_as_sorted_string() {
     if (strcmp(m, last_string) != 0) { // filter out duplicates
       st.print("%s%s", prefix, m);
       last_string = m;
-      prefix = "\n";
+      prefix = ",";
```

Comment:
Why did you change this back to ","?

Reply:
Including the "\n" causes the log messages to be a bit awkward, I think using a comma keeps it clear and on one line.